docs/faq.md

```diff
@@ -109,6 +120,29 @@ labels = query_human(ranks, pool)
 active_dataset.label(ranks, labels)
 ```
 
+## Theory FAQ
+
+Bayesian active learning is a relatively small field will a lot of unknowns. This sections aims at presenting some of our findings so that newcomers can quickly learn.
```

Comment:
```suggestion
Bayesian active learning is a relatively small field with a lot of unknowns. This section aims at presenting some of our findings so that newcomers can quickly learn.
```